build/dependenciesTest.props

```diff
@@ -7,6 +7,7 @@
     <MicrosoftNETTestSdkVersion>17.11.1</MicrosoftNETTestSdkVersion>
     <NetStandardVersion>2.0.3</NetStandardVersion>
     <NewtonsoftVersion>13.0.3</NewtonsoftVersion>
+    <OpenTelemetryVersion>1.10.0</OpenTelemetryVersion>
```

Comment:
I'd suggest starting with 1.6.0 that is used in other SDKs.
1.10.0 is still very new.